modules/openapi-generator/src/main/java/org/openapitools/codegen/CodegenModel.java

```diff
@@ -124,6 +124,7 @@ public class CodegenModel implements IJsonSchemaValidationProperties {
     @Getter @Setter
     public String arrayModelType;
     public boolean isAlias; // Is this effectively an alias of another simple type
+    public boolean isVariant; // Does this represent a schema variant?
```

Comment:
can you please add a comment showing an example of a schema variant to put everyone on the same page?